Makefile

```diff
@@ -36,28 +36,31 @@ INTEGRATION_TAGS?=integration
 E2E_TAGS?=e2e
 E2E_TIMEOUT?=60m
 
-export PATH := ./bin:$(PATH)
+export PATH := $(shell go env GOPATH)/bin:$(PATH)
+ifneq ($(OS),Windows_NT)
+	export SHELL := env PATH=$(PATH) /bin/bash
+endif
 export GO111MODULE := on
 export MCLI_E2E_BINARY
 export ATLAS_E2E_BINARY
 
-.PHONY: setupgolangcilint
-setupgolangcilint:  ## Install golangci-lint
-	@echo "==> Installing golangci-lint..."
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_VERSION)
-
 .PHONY: deps
 deps:  ## Download go module dependencies
 	@echo "==> Installing go.mod dependencies..."
 	go mod download
 	go mod tidy
 
-.PHONY: setup
-setup: deps setupgolangcilint ## Set up dev env
+.PHONY: devtools
+devtools:  ## Install dev tools
```

Comment:
Feel free to do as a follow up
https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md#environment

Reply:
@gssbzn I'm not too sure we need to change anything since `make setup` is still available.

I've only added `make devtools` as separated target because evergreen would benefit from it.